hubble_shuttle/tests/tests_shuttle_api.py

```diff
@@ -121,6 +121,13 @@ def test_get_404_http_error(self):
         self.assertEqual("ShuttleAPITestClient", cm.exception.service_name, "Sets the service name")
         self.assertEqual("/status/404", cm.exception.source, "Sets the error source")
 
+    def test_get_409_http_error(self):
+        with self.assertRaises(hubble_shuttle.exceptions.ConflictError) as cm:
+            ShuttleAPITestClient().http_get("/status/409")
+        self.assertEqual(409, cm.exception.internal_status_code, "Returns the error status code")
+        self.assertEqual("ShuttleAPITestClient", cm.exception.service_name, "Sets the service name")
+        self.assertEqual("/status/409", cm.exception.source, "Sets the error source")
```

Comment:
This test has the same name as the one below it - I suspect it _probably_ gets overwritten, so this test isn't running at all, but the second one is.

Reply:
The one below is 499

Reply:
omg i'm dumb

Reply:
Approved - sorry about that!

Reply:
no problem. we all need a laught sometimes 😂